### PR TITLE
Read me changes

### PR DESCRIPTION
**Pull request by tt-le**
- Removed IBM cloud instructions
- Added cloudnativetk instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,24 +26,30 @@ This application exposes the following endpoints:
...
+Install the IBM Garage for Cloud CLI.
 
-[IBM Cloud DevOps](https://www.ibm.com/cloud/devops) services provides toolchains as a set of tool integrations that support development, deployment, and operations tasks inside IBM Cloud.
+```$bash
+npm i -g @ibmgaragecloud/cloud-native-toolkit-cli
+```
+Use the IBM Garage for Cloud CLI to register the GIT Repo
+```$bash
+igc pipeline -n dev --tekton --pipeline ibm-appmod-liberty
+```
```
Comment (AustinSeto) [inline README.md:55]: When running this command, I get the following error:

```
Creating pipeline on openshift cluster in dev namespace
The target namespace does not exist
Create the namespace by running: igc namespace dev --dev
```

Should include mention of setting this up, perhaps including the suggested `igc namespace` command. Or maybe just use a placeholder and suggesting the user replace the namespace with their desired one.

Comment (AustinSeto) [inline README.md]: Get this issue on this set of commands

```
Retrieving available template pipelines from tools
Error registering crd:  Error: customresourcedefinitions.apiextensions.k8s.io "pipelines.tekton.dev" not found
    at /usr/local/lib/node_modules/@ibmgaragecloud/cloud-native-toolkit-cli/node_modules/kubernetes-client/backends/request/client.js:231:25
    at Request._callback (/usr/local/lib/node_modules/@ibmgaragecloud/cloud-native-toolkit-cli/node_modules/kubernetes-client/backends/request/client.js:168:14)
    at Request.self.callback (/usr/local/lib/node_modules/@ibmgaragecloud/cloud-native-toolkit-cli/node_modules/request/request.js:185:22)
    at Request.emit (events.js:314:20)
    at Request.<anonymous> (/usr/local/lib/node_modules/@ibmgaragecloud/cloud-native-toolkit-cli/node_modules/request/request.js:1154:10)
    at Request.emit (events.js:314:20)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/@ibmgaragecloud/cloud-native-toolkit-cli/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (events.js:420:28)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1244:12) {
  code: 404,
  statusCode: 404
}
```

Seems to be issue with node package. Checked with using `npm i -g @garage-catalyst/ibm-garage-cloud-cli` from Quarkus template, had to use `--force` to overwrite some stuff from `@ibmgaragecloud/cloud-native-toolkit-cli` and got a different error.

```
Error getting cluster_type from configMap `ibmcloud-config`. Attempting to retrieve it from the secret
Error getting cluster_type from secret `ibmcloud-apikey`. Defaulting to `kubernetes`
Getting git parameters
  Project git repo: https://github.com/tt-le/java-liberty-app.git
? Provide the branch the pipeline should use: dependencies
Setting up dev namespace
Checking for existing namespace: dev
Setting up pull secrets
Setting up tls secrets
Adding pull secrets to serviceAccount
Copying ConfigMaps
Copying Secrets
Copying Tekton tasks
Copying Tekton pipelines
Copying Jenkins credentials
Creating service account: pipeline
Creating Git PipelineResource
Creating Image PipelineResource
Error registering pipeline: configmaps "ibmcloud-config" not found
```

File: README.md [hunk 2]
```diff
@@ -26,24 +26,30 @@ This application exposes the following endpoints:
...
-### Deploying to IBM Cloud
+### Deploying 
+After you have created a new git repo from this git template, remember to rename the project. Edit the pom.xml and change the artifactId from the default name to the name you used to create the template.
 
-<p align="center">
-    <a href="https://cloud.ibm.com/developer/appservice/starter-kits/java-liberty-app">
-    <img src="https://cloud.ibm.com/devops/setup/deploy/button_x2.png" alt="Deploy to IBM Cloud">
-    </a>
-</p>
+Make sure you are logged into the IBM Cloud using the IBM Cloud CLI and have access to you development cluster. If you are using OpenShift make sure you have logged into the OpenShift CLI on the command line.
```
Comment (AustinSeto) [inline README.md]: Typo, change to "have access to **your** development cluster"

File: README.md [hunk 3]
```diff
@@ -26,24 +26,30 @@ This application exposes the following endpoints:
...
 
 The web application has a health endpoint which is accessible at `<host>:<port>/health`. The ports are set in the `pom.xml` file.
 
 ## Steps
 
 You can [deploy this application to IBM Cloud](https://cloud.ibm.com/developer/appservice/starter-kits/java-liberty-app) or [build it locally](#building-locally) by cloning this repo first.  Once your app is live, you can access the `/health` endpoint to build out your cloud native application.
 
-### Deploying to IBM Cloud
+### Deploying 
+After you have created a new git repo from this git template, remember to rename the project. Edit the pom.xml and change the artifactId from the default name to the name you used to create the template.
```
Comment (AustinSeto) [inline README.md]: Add backticks around `pom.xml`

File: README.md [hunk 4]
```diff
@@ -26,24 +26,30 @@ This application exposes the following endpoints:
...
 
-[IBM Cloud DevOps](https://www.ibm.com/cloud/devops) services provides toolchains as a set of tool integrations that support development, deployment, and operations tasks inside IBM Cloud.
+```$bash
+npm i -g @ibmgaragecloud/cloud-native-toolkit-cli
+```
+Use the IBM Garage for Cloud CLI to register the GIT Repo
+```$bash
+igc pipeline -n dev --tekton --pipeline ibm-appmod-liberty
+```
+See the **Deploy an app** guide in the [IBM Cloud-Native toolkit](https://cloudnativetoolkit.dev/) for details.
```
Comment (AustinSeto) [inline README.md]: Should clarify and say that this is under the "day 1" category at this URL. Possibly just say to go through the content under the entire "Day 1" category.

File: README.md (hunk 1 above)
Comment (AustinSeto) [reply README.md]: Regarding second error, found a potentially issue here which may provide explanation:
https://github.com/ibm-garage-cloud/ibm-garage-cloud-cli/issues/15

**Review by yeekangc — APPROVED** [review]

**Pull request merged**